registry/kusama.json

```diff
@@ -278,5 +278,13 @@
         "shortDescription": "Kusama validator.",
         "longDescription": "Japan",
         "isOrganization": false
+    },
+    {
+        "address": "DDCNPp8oeYBcBM44b32iSse4t4yfTnDJbjQxohF59Fo23EF",
+        "name": "Luke Schoen (ltfschoen)",
+        "image": "https://raw.githubusercontent.com/ltfschoen/opengov-delegate-registry/master/images/ltfschoen-image.png",
```

Comment:
Hello!
Thank you for your request, based on our rules all data should be in /novasamatech/opengov-delegate-registry/ , please, change it in order to get image from novasama repo

Reply:
thanks for the feedback, i've fixed that in this commit https://github.com/novasamatech/opengov-delegate-registry/pull/109/commits/13ec6d0be527f668a7ac42d69d42ef953a91cfeb